src/node_errors.h

```diff
@@ -60,6 +60,7 @@ void OnFatalError(const char* location, const char* message);
   V(ERR_INVALID_ARG_VALUE, TypeError)                                          \
   V(ERR_OSSL_EVP_INVALID_DIGEST, Error)                                        \
   V(ERR_INVALID_ARG_TYPE, TypeError)                                           \
+  V(ERR_INVALID_CHARACTER, RangeError)                                         \
```

Comment:
DOMException are expected in atob/btoa per https://html.spec.whatwg.org/multipage/webappapis.html#dom-atob

Reply:
It's wrapped in JavaScript side.

btw, this PR seems won't be merged according to @addaleax's suggestion.